### PR TITLE
[Feature][Zeta] Add operation retry params to seatunnel.yaml

### PR DESCRIPTION
**Pull request by EricJoy2048**

### CODE REVIEW

File: seatunnel-engine/seatunnel-engine-common/src/main/resources/seatunnel.yaml [hunk 1]
```diff
@@ -19,7 +19,9 @@ seatunnel:
     engine:
         backup-count: 1
         queue-type: blockingqueue
-        print-execution-info-interval: 60
+        print-execution-info-interval: 10
```
Comment (hailin0) [inline seatunnel-engine/seatunnel-engine-common/src/main/resources/seatunnel.yaml]: revert？

Comment (EricJoy2048) [reply seatunnel-engine/seatunnel-engine-common/src/main/resources/seatunnel.yaml]: done

**Comment by EricJoy2048** [comment]
@hailin0  PTAL again.

**Comment by ic4y** [comment]
@EricJoy2048 Do we still need this PR now?

**Pull request closed without merging**